docs/HowTo/GetStarted/Cakeshop.md

```diff
@@ -9,7 +9,7 @@
 
 * Download WAR file (Binary packages are available for macOS, Windows, and Linux platforms on the [releases](https://github.com/ConsenSys/cakeshop/releases) page.)
 * Run `java -jar cakeshop.war`
-* Navigate to [http://localhost:8080/](http://localhost:8080/)
+* Navigate to <http://localhost:8080/>
 
 !!!note
     when running in Windows, -Dgeth.node=geth must be specified as Quorum is not yet available on Windows OS*
```

Comment:
when running in Windows, `-Dgeth.node=geth` must be specified as Quorum is not yet available on Windows OS*